### DISCUSSION TITLE
Can swagger-codegen default package to Alamofire 5.x?

### DISCUSSION THREAD

**Discussion by BhupeshPruthi** [swagger-api/swagger-codegen#11135] [General]
When I use Swagger-codegen 3.x (UI tool or Terminal) to produce the package using yaml file for Swift 5, the Alamofire version is still 4.9.1. Is it possible to get the Swagger-codegen, which is compatible with Almaofire 5.x?

**Comment by waveChand** [comment]
Is any update on this matter?

**Reply by rptwsthi** [reply]
I guess not. I was facing the same issues and after downgrading Alamofire to 4.9.1 those errors went away.
